docker-compose.yml

```
@@ -37,7 +37,7 @@ services:
       - "5173:5173"
     environment:
       - CHOKIDAR_USEPOLLING=true
-      - VITE_API_URL=${VITE_API_URL}
+      - VITE_BACKEND_URL=${VITE_BACKEND_URL}

```

時間経ってて名前変えた方がいいことだけ覚えててちょっと間違った指摘しちゃった 🙏 


```suggestion
      - VITE_BACKEND_URL=${BACKEND_URL}
```
こうした方がいいんじゃないかってことでした！
指摘をそのまま受け入れるんじゃなくて一度考えてもらえると助かる！

vite側（front）で使う環境変数は `VITE_` っていう接頭辞が必要だけど、.env とかのdockerで使う環境変数には不要なので付けない方がいいんじゃないかってことです！
仮にviteを将来辞めた時、変数名にviteって残っちゃうので

`API_URL` を `BACKEND_URL` に変えたのは良さそう⭕️

---

参考
https://github.com/m-ouchi7/wedding_invitation/pull/23/changes#r2571487380